plonky2x/core/src/frontend/uint/uint32_n.rs

```diff
@@ -139,12 +139,12 @@ macro_rules! make_uint32_n {
                 let self_targets = self
                     .limbs
                     .iter()
-                    .map(|x| U32Target(x.0 .0))
+                    .map(|x| U32Target(x.variable.0))
```

Comment:
Is `U32Target` safe? Or is the reasoning here that this method is okay because we make other assumptions about what is going into the `U32Target`?

Reply:
Good catch.  I didn't go through any Target based structs in the plonky2x.